packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -905,6 +905,32 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
           'A ScrollController is required when Scrollbar.isAlwaysShown is true. '
           'Either Scrollbar.controller was not provided, or a PrimaryScrollController could not be found.',
         );
+        assert(
+          scrollController!.hasClients,
+          'The ScrollController has no positions attached. The Scrollbar cannot '
```

Comment:
Shouldn't this error message suggest what to do? This does not says what exactly a user did wrong and what to do to fix it.